src/sql/workbench/api/common/sqlExtHostTypes.ts

```diff
@@ -416,4 +416,39 @@ export interface INotebookManagerDetails {
 	handle: number;
 	hasContentManager: boolean;
 	hasServerManager: boolean;
+}
+
+export interface ISessionDetails {
```

Comment:
I wonder if this name is a bit generic and may have conflicts in the future.  i.e., do we think this is the only type of session we'll support, or should this be INotebookSessionDetails or something like that.  Same question for generic method shutdownSession, etc.?

Reply:
I can definitely rename these - in the sqlops file I segmented by namespace `nb` but am not doing it here. 

For the shutdownSession method it's on the ExtHostNotebookShape so I think that one should be fine. I'll review the non-namespaced interfaces to ensure it's clear what these are for.

Reply:
If they're namespaced in sqlops.d.ts it's probably fine as-is, since these APIs are internal details, collisions are unlikely.  I'm fine with whatever you do here.